README.md

```diff
@@ -19,8 +19,8 @@ The UI for the [AeroGear Data Sync Server](https://github.com/aerogear/data-sync
 
 ## Prerequisites
 
-There is a dependency on having a running postgres server. To run postgres in a docker container use the following command:
- 
+* A running PostgreSQL server. To run postgres in a docker container use the following command
```

Comment:
I think the `*` is not necessary but it's not a problem.

Reply:
Ah... it happens because the code format has 4 spaces instead of 3. See the fix in https://github.com/aerogear/data-sync-ui/pull/108 ;)